app/views/layouts/application.haml

```diff
@@ -148,6 +148,12 @@
           = link_to 'Contact Us', contact_path
           &bull;
           = link_to 'Contribute', contribute_path
+          &bull;
+          - news_count = News.num_unread_for(current_user)
+          = link_to news_index_path(page: (news_count > 1 ? news_count : nil)) do
```

Comment:
Should we store this away in a News method? News.latest_path_for(user) or something?